packages/components/select/__tests__/select.test.ts

```
@@ -4426,4 +4428,64 @@ describe('Select', () => {
+    const wrapEl = wrapper.find('.el-select-dropdown__wrap').element
+    const cleanup = [
+      defineGetter(wrapEl, 'clientHeight', 204),
+      defineGetter(wrapEl, 'scrollHeight', 500),
+    ]
+
+    try {
+      await makeScroll(wrapEl, 'scrollTop', 500)
+
+      expect(handleEndReached).toHaveBeenCalledWith('bottom')
+    } finally {
+      cleanup.forEach((fn) => fn())
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)